manifests/init.pp

```diff
@@ -0,0 +1,48 @@
+# @summary Profile for installing and managing Java.
+#
+# Profile for installing and managing Java.
+#
+# @param install_lock_files
+#   Boolean - Install lock files or not.
+#
+# @param lock_files
+#   Hash of lock files (and parent dir(s)).
+#
+# @param pkg_list
+#   Hash of package names to install
+#   Keys must be an OS Family name
+#   Values must be a Hash of the format:
+#       Key = package name
+#           A package name can be prefixed with "-" to remove it from the list.
+#           In this way, a higher level hiera layer can remove a package added
+#           by a lower, common layer.
+#
+#       Value (optional):
+#           1. <EMPTY> in which case, the package will be installed
+#           2. <HASH> conataining valid attributes for the 
+#              "package" native puppet resource type
```

Comment:
This docstrings comment needs updated, now that the key is not an OS Family name.

But, I don't see the point of this parameter continuing to be a Hash. I think it can be a simple array.

Reply:
Yep, changed to array (longer comment later i this is PR).

And I've updated the docstring and rebuilt REFERENCE.md.